### PR TITLE
feat: add subtractNum

### PR DESCRIPTION
**Pull request by willcaul**
https://github.com/char0n/ramda-adjunct/issues/483

### CODE REVIEW

File: src/subtractValue.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { flip, subtract } from 'ramda';
+
+/**
+ * Subtracts its first argument from its second argument.
+ *
+ * @func subtractValue
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @category Math
+ * @sig Number → Number → Number
```
Comment (char0n) [inline src/subtractValue.js]: Can we please replace with: `Number -> Number -> Number`. Although ramda is using `→` char we use `->` chars, something we can find on the keyboard ;]

File: src/subtractValue.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { flip, subtract } from 'ramda';
+
+/**
+ * Subtracts its first argument from its second argument.
+ *
+ * @func subtractValue
```
Comment (char0n) [inline src/subtractValue.js]: I'd opt for a name `subtractNum` to be consistent with future `divideNum` naming.

File: test/subtractValue.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import eq from './shared/eq';
+
+describe('subtractValue', function() {
+  it('should subtract the first number from the second number', function() {
+    eq(RA.subtractValue(3, 5), 2);
+    eq(RA.subtractValue(5, 3), -2);
+    eq(RA.subtractValue(3, 'foo'), NaN);
+  });
+
+  it('should curry', function() {
```
Comment (char0n) [inline test/subtractValue.js]:
```js
it('should curry', function() {
  eq(RA.subtractValue(3)(5), 2);
  eq(RA.subtractValue(3, 5), 2);
});

**Comment by willcaul** [comment]
@char0n - thanks for the PR comments... updated.

**Comment by char0n** [comment]
This is good to go

**Pull request merged**